From 6e8d33a52dc483bbf662c61eefca614991645af0 Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Tue, 18 Oct 2022 20:59:36 +0200
Subject: [PATCH 13/13] NXP-NFC AOSP add specific permissions

Patch the AOSP frameworks/native definitions to add specific permissions
---
 data/etc/android.sofware.nfc.beam.xml | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)
 create mode 100644 data/etc/android.sofware.nfc.beam.xml

diff --git a/data/etc/android.sofware.nfc.beam.xml b/data/etc/android.sofware.nfc.beam.xml
new file mode 100644
index 0000000000..b0a5e7ea73
--- /dev/null
+++ b/data/etc/android.sofware.nfc.beam.xml
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (C) 2013 The Android Open Source Project
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+
+<!-- This feature indicates that the device supports P2P -->
+<permissions>
+    <feature name="android.sofware.nfc.beam" />
+</permissions>
-- 
2.25.1

